scenarios/test_library_conf.py

```diff
@@ -0,0 +1,41 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, BaseTestCase, interfaces, released, bug, irrelevant, missing_feature
+from utils.tools import logger
+
+# PHP JAVA ok
+
+
+@irrelevant(library="cpp")
+class Test_HeaderTags(BaseTestCase):
+    """DD_TRACE_HEADER_TAGS env var support"""
+
+    @released(dotnet="2.1.0", golang="?", java="?", nodejs="?", php_appsec="?", python="?", ruby="?")
```

Comment:
`released` decorator is only valid on class. Basically: 

* one class <=> on feature
* N method => N use cases